### DISCUSSION TITLE
Is it possible to change a projects's rootPath in the projects.json file ...

### DISCUSSION THREAD

**Discussion by fade2gray** [alefragnani/vscode-project-manager#751] [Q&A] [answered]
... and retain the current state of the workspace, i.e. open editor tabs/files?

For example, I tried changing ...

`"rootPath": "c:\\Users\\username\\CEC\\CEC-w2g.code-workspace",`
to
`"rootPath": "c:\\Users\\username\\CEC-w2g\\CEC-w2g.code-workspace",`

... also changing the directory name from CEC to CEC-w2g in explorer, but when I closed and reopened VS Code it resulted in CEC-w2g workspace being empty.

Is it possible to achieve this?

**Comment by alefragnani** [comment, marked as answer]
Hi @fade2gray ,

Changing the `rootPath` attribute in `package.json` is totally possible. In fact, I not only create new entries, but also update those entries from time to time, when projects are moved in explorer. And the Project List on both, the Side Bar and Command Palette will be updated as well, a few seconds after saving `projects.json` file.

But, changing the `rootPath` attribute won't make the folder/file be moved/renamed automatically in the explorer. 

Hope this helps
